src/ripple/protocol/SystemParameters.h

```diff
@@ -42,7 +42,7 @@ systemName ()
 /** Number of drops in the genesis account. */
 constexpr
 XRPAmount
-INITIAL_XRP{ 100'000'000'000 * DROPS_PER_XRP };
+INITIAL_XRP{ static_cast<std::int64_t>(100'000'000'000) * DROPS_PER_XRP };
```

Comment:
Is this cast necessary? If it is, could you change it to a `safe_cast`?

Reply:
I don't think it's needed. Fixed in `c2ef2bb5d [fold] Remove unneeded static_cast`